prac_05/state_names.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
```

Comment:
I change these TODO notifications to 2DO after I'm done with them so pycharm doesn't flag them when commiting to Git